apps/ringtones/js/manage_template.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+/* global Tagged */
+(function(exports) {
+  exports.templates = {
```

Comment:
Please just use `var Templates = { ... };` here (at the global scope). That's more in keeping with the style for the rest of the ringtones app.